src/client/client-wrapper.ts

```diff
@@ -11,6 +11,16 @@ class ClientWrapper {
 
   constructor (page: Page, auth: grpc.Metadata) {
     this.client = page;
+    this.client.addListener('requestfinished', (request: Request) => {
+      this.client['__networkRequests'] = [];
```

Comment:
I believe this would wipe out the request array with each subsequent request.  Can mitigate with...

```javascript
this.client['__networkRequests'] = this.client['__networkRequests'] || [];
```